client/src/components/FavoriteCard.jsx

```diff
@@ -1,24 +1,12 @@
 import React from 'react';
 import Card from 'react-bootstrap/Card';
-import Button from 'react-bootstrap/Button';
-import * as ioicons from 'react-icons/io5'
 
-const FavoriteCard = ({favorite}) => {
-
-    const onUpdate = (toUpdateStudent) => {
-        toUpdate(toUpdateStudent)
-    }
-
-    const onDelete = (toDeleteStudent) => {
-        toDelete(toDeleteStudent)
-    }
+const FavoriteCard = ({ favorite }) => {
 
     return (
         <Card>
             <Card.Body>
-            <img src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${favorite.pokecode}.png`}/>
-            {/* <Button variant="outline-danger" onClick={()=>{onDelete(student)}} style={{padding: '0.6em', marginRight:'0.9em'}}><ioicons.IoTrash/></Button>
-            <Button variant="outline-info" onClick={()=>{onUpdate(student)}} style={{padding: '0.6em'}}> <ioicons.IoSync/></Button> */}
+                <img src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${favorite.pokecode}.png`} />
```

Comment:
Not a blocker, but if you want to improve accessibility, add the Pokemon's name as the alt text to this image.

Reply:
Thank you for the reminder, will do!